ui/lib/pki/addon/components/page/pki-certificate-details.hbs

```diff
@@ -5,10 +5,15 @@
 
 <Toolbar>
   <ToolbarActions>
-    <button type="button" class="toolbar-link" {{on "click" this.downloadCert}} data-test-pki-cert-download-button>
-      Download
-      <Chevron @direction="down" @isButton={{true}} />
-    </button>
+    <Hds::Button
+      @text="Download"
+      @color="secondary"
+      @icon="chevron-down"
+      @iconPosition="trailing"
```

Comment:
In toolbars we're leaving the the icon position as the default 'leading' unless the route navigates to a new place